### ISSUE TITLE
"invalid byte sequence in US-ASCII" for UTF-8 encoded Literals

### ISSUE THREAD

**Issue by rhdunn** [structured-data/linter#5]
The Reviews RDFa example (http://linter.structured-data.org/?url=http://linter.structured-data.org/examples/google-rs/review.rdfa.html) produces the following error:

```
ArgumentError: invalid byte sequence in US-ASCII
```

However, the document is marked as UTF-8 XML using the `<?xml?>` tag.

Also, it does not detect the meta charset attribute:

```
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8"/>
  <title>L’Amourita Pizza</title>
  <base href="http://example.org/"/>
</head>
<body>
<div xmlns:v="http://rdf.data-vocabulary.org/#" typeof="v:Review">
  <span property="v:itemreviewed">L’Amourita Pizza</span>
</div>
</body>
</html>
```

NOTE: This only occurs if a text node is being used as an RDF literal -- e.g. the `v:itemreviewed` span node above. If you remove that, the data is processed correctly, even if it contains UTF-8 characters elsewhere (e.g. in the `title` tag above).
